src/Sulu/Bundle/RouteBundle/Command/UpdateRouteCommand.php

```diff
@@ -32,6 +32,7 @@ protected function configure()
     {
         $this->setName('sulu:route:update')
             ->addArgument('entity', InputArgument::REQUIRED)
+            ->addArgument('locale', InputArgument::REQUIRED)
```

Comment:
Why is it required? Shouldn't it be possible to update all routes with omitting the locale?